corehq/apps/locations/ucr_expressions.py

```diff
@@ -62,3 +64,48 @@ def location_parent_id(spec, context):
         }
     }
     return ExpressionFactory.from_spec(spec, context)
+
+
+class AncestorLocationExpression(JsonObject):
+    """
+    For a given location id and location type, this expression returns the ancestor of the location that is the
+    given type.
+    If no such location exists, return None.
+    e.g. (boston.location_id, "state") => massachusetts.to_json()
```

Comment:
Thoughts on removing this comment? I think it would be better to have this kind of doc in one place (README.md). Very easy for htese to get out of sync